pkg/networkservice/common/updatepath/client.go

```diff
@@ -20,44 +20,35 @@ import (
 	"context"
 
 	"github.com/golang/protobuf/ptypes/empty"
-	"github.com/networkservicemesh/api/pkg/api/networkservice"
 	"google.golang.org/grpc"
 
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
-	"github.com/networkservicemesh/sdk/pkg/tools/token"
 )
 
-type updatePathClient struct {
-	commonUpdatePath
+type idClient struct {
+	name string
 }
 
-// NewClient - creates a NetworkServiceClient chain element to update the Connection.Path
-//             - name - the name of the NetworkServiceClient of which the chain element is part
-func NewClient(name string, tokenGenerator token.GeneratorFunc) networkservice.NetworkServiceClient {
-	return &updatePathClient{
-		commonUpdatePath: commonUpdatePath{
-			name:           name,
-			tokenGenerator: tokenGenerator,
-		},
-	}
+// NewClient - creates a new setId client.
+//             name - name of the client
+//             Iff the current pathSegment name != name && pathSegment.id != networkservice.Id, set a new uuid for
```

Comment:
```suggestion
//             If the current pathSegment name != name && pathSegment.id != networkservice.Id, set a new uuid for
```

Reply:
I guess the question here is: did @haiodo mean 'if and only if' (often abbreviated iff) and do we think the reader will understand iff as that if he did :)

Reply:
Thanks. Fixed.